tests/javaRosa.js

```diff
@@ -785,6 +785,7 @@ require([
                     <translation lang="en" default="">\
                         <text id="question1-label">\
                             <value>question1 en label</value>\
+                            <value form="markdown">question1 en help</value>\
```

Comment:
Why are we putting the help-text markdown content in the label itext item instead of in the help itext item? Where will we put label-text markdown content if at some point in the future we want to allow markdown in the label as well?